backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/shipment/M_InOut_StepDef.java

```diff
@@ -233,7 +254,9 @@ public void invokeGenerateShipmentsProcess(@NonNull final DataTable table)
 
 		final String quantityType = DataTableUtil.extractStringForColumnName(tableRow, ShipmentScheduleEnqueuer.ShipmentScheduleWorkPackageParameters.PARAM_QuantityType);
 		final boolean isCompleteShipments = DataTableUtil.extractBooleanForColumnName(tableRow, ShipmentScheduleEnqueuer.ShipmentScheduleWorkPackageParameters.PARAM_IsCompleteShipments);
-		final boolean isShipToday = DataTableUtil.extractBooleanForColumnName(tableRow, ShipmentScheduleEnqueuer.ShipmentScheduleWorkPackageParameters.PARAM_IsShipmentDateToday);
+		final boolean isShipToday = DataTableUtil.extractBooleanForColumnNameOr(tableRow, "IsShipToday", false);
+		final boolean isShipDateDeliveryDate = DataTableUtil.extractBooleanForColumnNameOr(tableRow, "OPT.IsShipDateDeliveryDate", false);
+		final LocalDate fixedShipmentDate = DataTableUtil.extractLocalDateOrNullForColumnName(tableRow, "OPT." + ShipmentScheduleEnqueuer.ShipmentScheduleWorkPackageParameters.PARAM_FixedShipmentDate);
```

Comment:
so now that u modified the whole method, why not change it and use DataTableRows instead?